content/en/cloud_cost_management/container_cost_allocation.md

```diff
@@ -193,19 +200,19 @@ Use the `allocated_spend_type` tag to visualize the spend category associated wi
 
 ### Compute
 
-The cost of a host instance is split into two components: 60% for the CPU and 40% for the memory. Each component is allocated to individual workloads based on their resource reservations and usage.
+The cost of a host instance is split into two components: 60% for the CPU and 40% for the memory. If there is GPU usage on the pod, the cost of the host instance is split into three components: 95% for the GPU, 3% for the CPU, and 2% for the memory. Each component is allocated to individual workloads based on their resource reservations and usage.
 
 Costs are allocated into the following spend types:
 
 | Spend type | Description    |
 | -----------| -----------    |
-| Usage | Cost of resources (such as memory and CPU) used by workloads, based on the average usage on that day. |
-| Workload idle | Cost of resources (such as memory and CPU) that are reserved and allocated but not used by workloads. This is the difference between the total resources requested and the average usage. |
-| Cluster idle | Cost of resources (such as memory and CPU) that are not reserved by workloads in a cluster. This is the difference between the total cost of the resources and what is allocated to workloads. |
+| Usage | Cost of resources (such as memory, CPU, and GPU) used by workloads, based on the average usage on that day. |
+| Workload idle | Cost of resources (such as memory, CPU, and GPU) that are reserved and allocated but not used by workloads. This is the difference between the total resources requested and the average usage. |
+| Cluster idle | Cost of resources (such as memory, CPU, and GPU) that are not reserved by workloads in a cluster. This is the difference between the total cost of the resources and what is allocated to workloads. |
 
 ### Persistent volume
 
-The cost of an AWS EBS volume has three components: IOPS, throughput, and storage. Each is allocated according to a pod's usage when the volume is mounted.
+The cost of an EBS volume has three components: IOPS, throughput, and storage. Each is allocated according to a pod's usage when the volume is mounted.
```

Comment:
Any reason why you made this change?

Reply:
was failing the vale check, reason was it did not approve of having 'AWS EBS' so maybe new linter rules were added?